src/pages/view-test-result/cat-a-mod2/components/debrief-card/debrief-card.ts

```diff
@@ -78,41 +62,37 @@ export class DebriefCardComponent {
   }
 
   public getDrivingFaults(): FaultSummary[] {
-    return this.faultSummaryProvider.getDrivingFaultsList(this.data, TestCategory.BE);
+    return this.faultSummaryProvider.getDrivingFaultsList(this.data, TestCategory.EUAM1);
```

Comment:
M1? should be M2 no?

Reply:
Ahh just saw Tom's comment - ignore this if you're passing the category data

Reply:
Thanks Szab.